ReactQt/runtime/src/reactuimanager.cpp

```diff
@@ -120,6 +120,20 @@ void ReactUIManager::updateView
   m_bridge->visualParent()->polish();
 }
 
+void ReactUIManager::setChildren
+(
+  int containerReactTag,
+  const QList<int>& childrenTags
+)
+{
+  //This is a simple implementation which fixes a broken example. It's not properly tested and may need revisiting
+  QList<int> newIndices;
+  foreach(int i, childrenTags) {
+     newIndices.append(i + 1);
```

Comment:
@narek-h , since `newIndices` are used in `manageChildren` to set Z-order on displayed `QQuickItem`, it might be not correct to increment Z-order for every child. If to take TicTacToe app, as example, displayed qml items cells most of all should have the same Z-order value.